test/controllers/api/v2/host_tracer_controller_test.rb

```diff
@@ -15,10 +15,59 @@ def setup
     end
 
     def test_index
-      results = JSON.parse(get(:index, params: { :host_id => @host.id }).body)
+      @host1.host_traces.create!(:id => 1, :host_id => 1, :helper => 'agile', :app_type => 'foo', :application => 'scrumm')
+
+      results = JSON.parse(get(:index, params: { :host_id => @host1.id }).body)
+
+      assert_response :success
+      assert_includes results['results'].collect { |item| item['id'] }, @host1.host_traces.first.id
+    end
+
+    def test_resolve_group_by_helper
```

Comment:
These new tests fail on master because the 'JobInvocationComposer' class doesn't exist since we don't have REX there. We can easily work around this by defining the class ourselves near the top of this file - under the `requires`  - if REX is not present, like this:

```
unless Katello.with_remote_execution?                                                                                                                                                         
  class JobInvocationComposer                                                                                                                                                                 
  end                                                                                                                                                                                         
end
```